components/aws/sagemaker/tests/integration_tests/component_tests/test_deploy_component.py

```diff
@@ -12,6 +12,14 @@
 from utils import sagemaker_utils
 
 
+@pytest.mark.parametrize(
```

Comment:
this shouldn't be moved. `run_predict_mnist` is a helper method and `test_create_endpoint` is the actual test